app/src/main/java/com/example/workoutapp/RetrofitClientInstance.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.workoutapp
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClientInstance {
+    private var retrofit: Retrofit? = null
+    private val BASE_URL = ""
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere? 

Recommendation: if not used, remove it, or keep it in a separate development branch until it is complete.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.